bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Why all this?

`env` is not even guaranteed to be in /usr/bin

There is no good reason for using `#!/usr/bin/env bash` - either your script must have bash, in which case you are assuming it is installed so it is in /bin - or, your script can get by without bash with just Bourne shell - in which case every UNIX will have /bin/sh.